CONTRIBUTING.md

```diff
@@ -144,12 +144,15 @@ We follow the [official go guidelines](https://github.com/golang/go/wiki/CodeRev
 | :------------------: | :-----: | :-----------------------------------------------------------: |
 | context              | ctx     | context.Context                                               |
 | mutex                | mu      | sync.Mutex                                                    |
-| configuration        | conf    | go.thethings.network/lorawan-stack/pkg/config.Config             |
-| logger               | logger  | go.thethings.network/lorawan-stack/pkg/log.Logger                |
-| message              | msg     | go.thethings.network/lorawan-stack/api/gateway.UplinkMessage     |
-| status               | st      | go.thethings.network/lorawan-stack/api/gateway.Status            |
-| server               | srv     | go.thethings.network/lorawan-stack/pkg/network-server.Server     |
-| EUI                  | eui     | go.thethings.network/lorawan-stack/pkg/types.DevEUI              |
```

Comment:
Let's not expand those EUIs here, we already have them below. A `see below` would be sufficient.